apps/nextjs/src/app/_components/landing/header/header.tsx

```
@@ -20,49 +19,49 @@ function Header() {
     <header className="fixed inset-x-0 top-0 z-10 bg-background/80 backdrop-blur-lg">
       <div className="flex h-16 items-center justify-between px-4 md:px-6">
         <div className="flex w-full justify-between">
-          <Link href="/" className="flex items-center justify-center">
-            <NotebookTextIcon className="h-6 w-6" />
-            <span className="sr-only">Posts Buddy</span>
+          <Link href="/" className="flex items-center justify-center gap-2">
+            <BuildingIcon className="h-6 w-6" />
+            <span className="font-semibold">Enterprise Suite</span>
           </Link>
 
           <button className="md:hidden" onClick={toggleMenu}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)